src/main/webapp/javascript/user-page.js

```diff
@@ -1,9 +1,29 @@
+// loads all the recipes when the recipe loads
+async function loadRecipes() {
```

Comment:
Is there a reason there are two functions instead of just one which loads all the recipes?

Reply:
no real reason I just think it'll be neater since I'll have to call different servlets and stuff